doc/Makefile

```diff
@@ -40,6 +40,7 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)/*
+	-rm -rf source/api/_as_gen/*
```

Comment:
Should the 'gallery' and 'api' dirs also be removed? (what do they do in other projects?)

Reply:
I usually don't remove the gallery dir since it can take a long time to generate (if the examples take a while to build). If I want that gone, then I just manually delete the folder. But LMK what you'd prefer!